share/p2p/shrexeds/client.go

```diff
@@ -119,6 +125,8 @@ func (c *Client) doRequest(
 
 	switch resp.Status {
```

Comment:
Make sure to update read deadline after you read status. You need bigger timeout for ods read

Reply:
my bad, found the line